Controllers/CustumerApiController.cs

```diff
@@ -19,7 +26,7 @@ public IActionResult SearchAirports(string search)
 
             if (validValues.Contains(searchTerm))
             {
-                return Ok(new[] { new { airport = "RIX", city = "Riga", country = "Latvia" } });
+                 return Ok(_context.Flights.FirstOrDefault(x => x.To.AirportCode == "RIX"));
```

Comment:
nu šis joprojām nav īsti pareizi... un kas tas par validValues? 
kapēc joprojām meklē pēc hardkodētas vērtības? nevis pēc tā kas tiek prasīts? 
un kapēc no db meklē caur flights? tev taču ir atsevišķa Airports tabula tur..